packages/event-handler/src/rest/ErrorHandlerRegistry.ts

```diff
@@ -71,4 +71,19 @@ export class ErrorHandlerRegistry {
 
     return null;
   }
+
+  /**
+   * Merges another {@link ErrorHandlerRegistry | `ErrorHandlerRegistry`} instance into the current instance.
+   * It takes the handlers from the provided registry and adds them to the current registry.
+   *
+   * @param errorHandlerRegistry - The registry instance to merge with the current instance
+   */
+  public merge(errorHandlerRegistry: ErrorHandlerRegistry): void {
```

Comment:
As with routes, we should call out that last one wins here in the case of error handlers associated with the same `Error` constructor that already exist in the registry.